### DISCUSSION TITLE
WikiFolderUpdateTask Upload Folders in multi file mode

### DISCUSSION THREAD

**Discussion by daandupau** [rfennell/AzurePipelines#1287] [Q&A]
Is it possible to update folders recursively while only looking at *.md files and creating the folders remotely?

should i add `<Directory>/*.md`  to the filter field would that work?

**Comment by daandupau** [comment]
i see that the default is: `**/*.md` 

we have this: `*.md` currently

what would happen if i use `**/*.md`

would it also upload .csv files or would it skip those?

**Reply by rfennell** [reply]
`**/*.md` means search for `*.md` files recursively from the current working folder. It will only find .MD file

**Reply by daandupau** [reply]
thanks :) this issue can be closed
